google/cloud/speech/cloud_speech.yaml

```diff
@@ -11,17 +11,18 @@ documentation:
     Google Cloud Speech API.
 
 apis:
-- name: google.cloud.speech.v1.Speech
+- name: google.cloud.speech.v1beta1.Speech
+- name: google.longrunning.Operations
```

Comment:
Agreed. It is still unclear to us about the long running operations.
Can we exclude AsyncRecognize and longrunning.Operations for the time being?